zipline-gradle-plugin/src/test/projects/no-runtime/lib/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2024 Cash App
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import app.cash.zipline.gradle.ZiplineCompileTask
+import org.jetbrains.kotlin.gradle.targets.js.yarn.YarnPlugin
+import org.jetbrains.kotlin.gradle.targets.js.yarn.YarnRootExtension
+
+plugins {
+  kotlin("multiplatform")
+  id("app.cash.zipline")
```

Comment:
Will this add the runtime unconditionally? I don’t know how to be confident that this won’t regress through automatic dependencies, other than perhaps explicitly excluding it

Reply:
The plugin does not add the runtime today. This test would continue to pass if it did, though. I can add an assertion against that.

Reply:
Huh I added a test for that and something unrelated failed, so I'll be fixing two things!

Reply:
Done